From 1283990f8feccfbbfe10f64d319c53324eeb7bf6 Mon Sep 17 00:00:00 2001
From: William Douglas <william.douglas@intel.com>
Date: Fri, 10 Aug 2018 10:39:01 -0500
Subject: [PATCH 1/2 v2] Update default crio.conf file for Clear Linux

Update paths to conmon and cni plugins to reflect where they are
intended to be used from Clear Linux. Update configuration to use
Clear Containers as the untrusted workload handler and that untrusted
workloads are the default workload type.

Signed-off-by: Jose Lamego <jose.a.lamego@intel.com>
---

Notes:
    Changes in v2: Updated path to seccomp.json and cni plugin directory.
            This change restores out-of-the-box crio service functionality.

 Makefile | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/Makefile b/Makefile
index 82b8e77..503453e 100644
--- a/Makefile
+++ b/Makefile
@@ -95,6 +95,11 @@ bin/crio: .gopathok
 
 crio.conf: bin/crio
 	./bin/crio --config="" config --default > crio.conf
+	sed -i 's|^\(conmon = \).*|\1"/usr/libexec/crio/conmon"|' crio.conf
+	sed -i 's|^\(plugin_dir = \).*|\1"/usr/libexec/cni/"|' crio.conf
+	sed -i 's|^\(runtime_untrusted_workload = \).*|\1"/usr/bin/cc-runtime"|' crio.conf
+	sed -i 's|^\(default_workload_trust = \).*|\1"untrusted"|' crio.conf
+	sed -i 's|^\(seccomp_profile = \).*|\1"/usr/share/defaults/crio/seccomp.json"|' crio.conf
 
 release-note:
 	@$(GOPATH)/bin/containerd-release -n $(release)
-- 
2.18.0

